simpleui-server/src/sui_zmq.ts

```diff
@@ -49,16 +49,13 @@ export class ZmqSocket {
             let zmqRequestPacket = "";
 
             if (req.method === 'POST') {
-                console.log('--- got CMD')
                 zmqRequestPacket = SuiData.buildZmqCmdPacket(req);
             }
             else {
                 zmqRequestPacket = SuiData.buildZmqDataPacket(req);
             }
             // send request
             this.outboundMessages += 1;
-            if (zmqRequestPacket.includes("cmd="))
-                console.log(`Sending ZMQ: ${zmqRequestPacket}`)
             this.socket.send(zmqRequestPacket);
```

Comment:
Should you only send when `(!cmdLine.includes('--readonly')` to make sure somebody isn't cleverly manipulating with javascript?

Reply:
I think this is a good idea to add more redundancy. I just added it